app/web/src/newhotness/logic_composables/change_set.ts

```diff
@@ -42,14 +79,11 @@ const useDisableApplyChangeSetInner = () => {
   const ctx = inject<Context>("CONTEXT");
   assertIsDefined(ctx);
 
-  const currentChangeSet = useCurrentChangeSet();
   const status = useStatus();
 
-  const changeSet = computed(() => currentChangeSet.value);
-
   return computed(
     () =>
-      changeSet.value?.status !== ChangeSetStatus.Open ||
+      ctx.changeSet.value?.status !== ChangeSetStatus.Open ||
```

Comment:
Now, the `Context` has the `changeSet` the user is on, so we don't need to re-use the other composable